drivers/nrf24l01p/nrf24l01p.c

```diff
@@ -618,30 +618,13 @@ int nrf24l01p_set_power(const nrf24l01p_t *dev, int pwr)
     return nrf24l01p_write_reg(dev, REG_RF_SETUP, rf_setup);
 }
 
+static const int _nrf24l01p_power_map[4] = { -18, -12, -6, 0 };
```

Comment:
You can save a few bytes and use `int8_t[]` here.

Reply:
I tested and this makes no difference in code size. Anyway I think it's better to be explicit and use int8_t instead of int.
I pushed the change, directly squashed.